bazarr/app/logger.py

```diff
@@ -161,7 +164,7 @@ def configure_logging(debug=False):
         logging.getLogger("websocket").setLevel(logging.CRITICAL)
         logging.getLogger("ga4mp.ga4mp").setLevel(logging.ERROR)
 
-    logging.getLogger("waitress").setLevel(logging.ERROR)
+    logging.getLogger("waitress").setLevel(logging.INFO)
```

Comment:
On purpose this change is made to make no net difference in the end, see lines 65 and 66. The filter applied to the waitress logger still filters out anything that isn't logging at ERROR or higher.  

The reason I moved the log level filtering from the global level setting into the filter is because the waitress function to log listen information (`print_listen`) uses INFO as the log level. With a level set to ERROR, this would never show up. Instead I moved the filtering details into the existing custom logging filter for waitress and used 'BAZARR' as a keyword to allow the message to pass (line 61). Everything else is still blocked if it's not at least ERROR level or higher (lines 65 and 66).  
See also the change in server.py and the comment there.  

If there is a better way to control the logging in this regard I would love to hear it but the waitress function did not seem to expose an option to supply a preferred level so this seemed like the only way to do this.  
If it's OK for waitress to just be in INFO mode then I could just remove the changes on lines 61, 65 and 66.